### ISSUE TITLE
A message adds after cliking on enter even if its empty

### ISSUE THREAD

**Issue by savayer** [MatheusrdSantos/vue-quick-chat#42]
I tried
```
async onMessageSubmit (message) {
  message.content = message.content.trim().replace(/\r\n/g, '')
  if (message.content === '') {
    return
  }
  ...
```

but empty message still adds
it seems newMessage mutation should have some condition

can you fix it? Or is there a way how can i avoid it?

**Comment by MatheusrdSantos** [comment]
Hi, @savayer. I'll fix it tomorrow. Probably I forgot to check the message content.

**Comment by MatheusrdSantos** [comment]
Hi, @savayer. It's fixed now 962b2d146faba6cdfdefc205724c9d01ed05e621. Just update to version 1.2.6.
Thanks for your report :)

**Comment by savayer** [comment]
Great, thanks!
